### DISCUSSION TITLE
graphscope使用中的建议

### DISCUSSION THREAD

**Discussion by thinkdnt** [alibaba/GraphScope#2705] [General]
有几点建议：
1、香港镜像地址有时候在 aws ec2 上拉取超时，是否考虑可以放到国内？
比如：registry.cn-hongkong.aliyuncs.com -> registry.cn-hangzhou.aliyuncs.com
2、感觉大部分的交互和原理都属于黑盒，用户只能一键使用和调API，如果步骤中出现任何报错, 都容易迷茫或手足无措。
期待能够完善下相关文档，比如整体架构及各个模块的介绍应用，这样排查问题也相对容易点。
3、ubuntu镜像是否能把基础工具封装进去，有时候容器中无法联网安装，排查故障非常不方便。
比如dnsutils(dig,ping,nslookup)等？
4、官方文档中常见问题中钉钉已经失效，请更新下，要不然联系不到技术支持。

以上几点只是目前碰到的问题，最后感谢开发者李东泽帮助排查问题。

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! And a maintainer will get back to you shortly!
Please feel free to contact us on [DingTalk](https://h5.dingtalk.com/circle/healthCheckin.html?dtaction=os&corpId=dingb4b76e0fa9ccd8d516fd0f0acdf19f55&68e71190=0fe34ec0&cbdbhh=qwertyuiop), WeChat account(graphscope) or [Slack](http://slack.graphscope.io/). We are happy to answer your questions responsively.

**Comment by yecol** [comment]
Hi @thinkdnt, 

Many thanks to your nice suggestions!
We are going to refine the documentations on Nov. and Dec. 
All your suggestions will be taken and corresponding actions will be made!
